### PR TITLE
add the div fct

### PR DESCRIPTION
**Pull request by ana-beatrice**
added div fct

### CODE REVIEW

File: program.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+def div(a,b):
+    return a/b
```
Comment (florinIuga) [inline program.py:8]: I think that you didn't consider the case when b is equal to 0. Could you add a check for that please?

Comment (ana-beatrice) [reply program.py:8]: Sure I've forgot about it.I will update there pr in a sec.Thanks

Comment (ana-beatrice) [reply program.py:8]: - [x] Done

**Review by florinIuga — APPROVED** [review]
Looks good to me

**Pull request merged**